tests/fixtures/tests.py

```diff
@@ -688,6 +688,21 @@ def test_exclude_option_errors(self):
         with self.assertRaisesMessage(management.CommandError, msg):
             management.call_command('loaddata', 'fixture1', exclude=['fixtures.FooModel'], verbosity=0)
 
+    def test_natural_keys_with_multi_table_inheritance_models(self):
```

Comment:
Did you consider `tests/serializers` as a location for the test?

No need to test other formats, you think?